python_modules/libraries/dagstermill/dagstermill_tests/test_logging.py

```diff
@@ -30,7 +30,16 @@ def __init__(self, file_path):
 
     def emit(self, record):
         with open(self.file_path, "a", encoding="utf8") as fd:
-            fd.write(seven.json.dumps(record.__dict__) + "\n")
+            fd.write(
+                seven.json.dumps(
+                    {
+                        "dagster_meta": {
+                            "orig_message": record.__dict__["dagster_meta"]["orig_message"]
+                        }
+                    }
```

Comment:
how did you end up with this? leave a comment at least

Reply:
was trying to fit the asserts below, but we don't actually need this nested structure. I changed to simplify.